### DISCUSSION TITLE
systemverilog.excludeIndexing vs files.exclude

### DISCUSSION THREAD

**Discussion by sarki94g** [eirikpre/VSCode-SystemVerilog#143] [Q&A]
I know how **files.exclude** works in that it will reduce what is shown in the directory tree.  Question is, does **systemverilog.excludeIndexing** ignore **files.exclude** or use it as a starting point?  

Reason I ask is because the "Go to Definition" on a module instance is opening files eliminated by **files.exclude**.

**Comment by eirikpre** [comment]
That might be the case that files.exclude is ignored.

If files.exclude is updated runtime (after indexing has run) the extension will most certainly not delete the already indexed entry
